components/syntax_analyzer.py

```diff
@@ -484,22 +484,22 @@ def __r17_assign(self):
             text_2 = f"Expected `=`, but found {self.__current_token.lexeme}"
             raise SyntaxError(f"{text_1}\n{text_2}")
 
-    def __r18a_if(self):
+   def __r18a_if(self):
```

Comment:
Indentation Error: unindent does not match any outer indentation level.
Here is my suggestion:
```suggestion
    def __r18a_if(self):
```